site/docs/use/00-user.md

```diff
@@ -8,20 +8,127 @@ title: User Manual
 
 ### Self Register User
 
+Zitadel allows users to register a organization and/or user with just a few steps.
+
+1. Register a organization
+2. Verify your email
+3. Login to Zitadel and manage the organization
+
+
+<div class="zitadel-gallery" itemscope itemtype="http://schema.org/ImageGallery">
+    <figure itemprop="associatedMedia" itemscope itemtype="http://schema.org/ImageObject">
+        <a href="img/register.gif" itemprop="contentUrl" data-size="1100x906">
+            <img src="img/register.gif" itemprop="thumbnail" alt="Self Register" />
+        </a>
+        <figcaption itemprop="caption description">Self Register</figcaption>
+    </figure>
+</div>
+
+
 ### Verify EMail
 
+To verify our email address just klick the "Finish Initialization" link in the email your received after registration. You could copy and paste the received code as well and enter it at the initial login.
+
+<div class="zitadel-gallery" itemscope itemtype="http://schema.org/ImageGallery">
+    <figure itemprop="associatedMedia" itemscope itemtype="http://schema.org/ImageObject">
+        <a href="img/email-verify.gif" itemprop="contentUrl" data-size="1100x906">
+            <img src="img/email-verify.gif" itemprop="thumbnail" alt="Verify EMail" />
+        </a>
+        <figcaption itemprop="caption description">Verify EMail</figcaption>
+    </figure>
+</div>
+
+
 ### Verify Phone
 
+tbd
+
 ### Change Password
 
+To change your password you can hit the link right at the overview page. Alternatively  you can set it in the "Personal Information" page.
+
+
+
+<div class="zitadel-gallery" itemscope itemtype="http://schema.org/ImageGallery">
+    <figure itemprop="associatedMedia" itemscope itemtype="http://schema.org/ImageObject">
+        <a href="img/change-password.gif" itemprop="contentUrl" data-size="1100x906">
+            <img src="img/change-password.gif" itemprop="thumbnail" alt="Change Password" />
+        </a>
+        <figcaption itemprop="caption description">Change Password</figcaption>
+    </figure>
+</div>
+
 ### Manage Multi Factor
 
+To enable multifactor authentication visit the "Personal Information" page of your account and scroll to the "multifactor authentication". 
+You can either:
+
+1. Configure OTP
+2. AddU2F
+
+
+<div class="zitadel-gallery" itemscope itemtype="http://schema.org/ImageGallery">
+    <figure itemprop="associatedMedia" itemscope itemtype="http://schema.org/ImageObject">
+        <a href="img/enable-mfa-handling.gif" itemprop="contentUrl" data-size="1100x906">
+            <img src="img/enable-mfa-handling.gif" itemprop="thumbnail" alt="Enable Multi Factor" />
+        </a>
+        <figcaption itemprop="caption description">Encale Multi Factor</figcaption>
```

Comment:
What is encale?

Reply:
a dynamic word for "enable",... damn typos